components/Header/MobileNavbar.tsx

```diff
@@ -10,15 +10,13 @@ import { headerNavbarLinks } from "./constants";
 const navbarInitialMotionStyle = {
   top: 0,
   right: 0,
-  opacity: 0,
```

Comment:
Rollback these changes - I was testing the navbar without transitioning the opacity but it isn't worth changing this here.